tests/differentiate.py

```diff
@@ -0,0 +1,43 @@
+from argparse import ArgumentParser
```

Comment:
This script needs to have a docstring explaining what is it's purpose. Also I think it may be better to rename it to something more self-explicable.

For what I understand this test was used to generate the diff IO tests, so it should probably end up in there, unless there is a more general use for it.

Reply:
It was used to produce a bitstreams which are technically compatible with ZCU104 platform as it has only differential clock inputs.